projects/forge-angular/src/lib/app-bar/app-bar-router-navigate.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, ElementRef } from '@angular/core';
+import { Router } from '@angular/router';
+import { IAppBarComponent } from '@tylertech/forge';
+
+@Directive({
+  selector: 'forge-app-bar[href]'
+})
+export class AppBarRouterNavigateDirective {
```

Comment:
This was just a prototype for now, and it will get automatically applied since the `<a>` is within the shadow DOM of the app bar component so it "just works" without anyone needing to know about it. I can add some comments to it though so it's clear what it is used for.